modin/pandas/test/test_io.py

```diff
@@ -1834,6 +1843,32 @@ def test_read_sql(self, make_sql_connection):
         pandas_df = pandas.read_sql(sql=query, con=sqlalchemy_connection)
         df_equals(modin_df, pandas_df)
 
+    @pytest.mark.skipif(
```

Comment:
I remember we wanted to avoid skip in favor of xfail? Should this be the latter?